doc/sphinxext/redirect_from.py

```diff
@@ -15,6 +15,7 @@
 This creates in the build directory a file ``build/html/topic/old-page.html``
 that contains a relative refresh::
 
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
```

Comment:
The `meta` tag should be in the `head` tag, not top-level.

Reply:
Oopsie.  I don't claim any understanding of the html standard; do you know the correct syntax to make this interact with the other meta tag?

Reply:
They're independent, it just needs to be moved down, but also could be charset instead of a content-type: #23030